geniza/corpus/models.py

```diff
@@ -525,6 +525,22 @@ def editions(self):
             "content", "source"
         )
 
+    def digital_editions(self):
+        """All footnotes for this document where the document relation includes
+        edition AND the footnote has content."""
+        return (
+            self.footnotes.filter(doc_relation__contains=Footnote.EDITION)
+            .filter(content__isnull=False)
+            .order_by("content", "source")
```

Comment:
I'm not sure how much sense this sorting makes, but not sure what other option to suggest.